packages/grafana-perf-tests/webpack.config.js

```diff
@@ -0,0 +1,50 @@
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyPlugin = require('copy-webpack-plugin');
+const path = require('path');
+const GlobEntries = require('webpack-glob-entries');
+
+module.exports = {
+  mode: 'production',
+  entry: GlobEntries('./src/*test*.ts'), // Generates multiple entry for each test
+  output: {
+    path: path.join(__dirname, 'dist'),
+    libraryTarget: 'commonjs',
+    filename: '[name].js',
+  },
+  resolve: {
+    extensions: ['.ts', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.ts$/,
+        use: 'babel-loader',
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  target: 'web',
+  externals: /^(k6|https?\:\/\/)(\/.*)?/,
+  // Generate map files for compiled scripts
+  devtool: 'source-map',
+  stats: {
+    colors: true,
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
+    // Copy assets to the destination folder
+    // see `src/post-file-test.ts` for an test example using an asset
+    new CopyPlugin({
+      patterns: [
+        {
+          from: path.resolve(__dirname, 'assets'),
```

Comment:
Is this still required? I cannot see an assets directory or `src/post-file-test.ts` in this PR.

Reply:
removed for now - will bring back together with publishing in a subsequent pr